src/deleteAllDocs.js

```diff
@@ -0,0 +1,17 @@
+const { db } = require('./connect')();
+
+db.list()
+.then(res => {
+    let docs = [];
+
+    for (let x of res.rows) {
+        let data = {};
+        data._id = x.id;
+        data._rev = x.value.rev;
+        data._deleted = true;
```

Comment:
The more javascriptey way to do this would be:
```js
const data = {
	_id = x.id,
	_rev = x.value.rev,
	_deleted = true
};
```
two things: use `const` if you don't change the variable after assignement, and create the object in one go.